backend/app/api/endpoints/openapi_responses.py

```
@@ -294,95 +294,481 @@ async def create_response(
+        # Dispatch task asynchronously via HTTP+Callback mode
+        # The executor_manager will execute the task and send results via callback
+        import asyncio
 
-    # Get subtasks for output
-    subtasks = (
-        db.query(Subtask)
-        .filter(
-            Subtask.task_id == task_dict.get("id"), Subtask.user_id == current_user.id
+        asyncio.create_task(
+            execution_dispatcher.dispatch(execution_request, emitter=None)
+        )
```

Ruff (0.14.14)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)